passes/techmap/Makefile.inc

```diff
@@ -13,6 +13,7 @@ OBJS += passes/techmap/abc9_ops.o
 ifneq ($(ABCEXTERNAL),)
 passes/techmap/abc.o: CXXFLAGS += -DABCEXTERNAL='"$(ABCEXTERNAL)"'
 passes/techmap/abc9.o: CXXFLAGS += -DABCEXTERNAL='"$(ABCEXTERNAL)"'
```

Comment:
Do we still need this line?

Reply:
Do you mean the one for `abc9.o`? There still is an `#ifdef ABCEXTERNAL` in there, but only for the `help()` message. So in absolute terms, the answer is "yes", unless you can (or want/need to) remove the remaining `#ifdef` from `techmap/abc9.cc`.

Reply:
Good thinking. I forgot we had that line!